src/commands/Minion/buy.ts

```diff
@@ -33,25 +33,40 @@ export default class extends BotCommand {
 
 		await msg.author.settings.sync(true);
 		const GP = msg.author.settings.get(UserSettings.GP);
-		if (GP < buyable.gpCost) {
-			throw `You need ${toKMB(buyable.gpCost)} GP to purchase this item.`;
-		}
 		const QP = msg.author.settings.get(UserSettings.QP);
-		if (QP < buyable.qpRequired) {
-			throw `You need ${buyable.qpRequired} QP to purchase this item.`;
-		}
-
+		const titheFarmPoints = msg.author.settings.get(UserSettings.Stats.TitheFarmPoints);
 		const itemString = await createReadableItemListFromBank(this.client, buyable.outputItems);
 
-		if (!msg.flagArgs.cf && !msg.flagArgs.confirm) {
-			const sellMsg = await msg.channel.send(
-				`${
-					msg.author
-				}, say \`confirm\` to confirm that you want to purchase ${itemString} for ${toKMB(
-					buyable.gpCost
-				)}.`
-			);
+		let sellMsg: any;
```

Comment:
dont use `any`! give this a type. because youre declaring the variable but not immediately assigning a value, you will want to do `type | undefined` where `type` is the type it should be, in this case I think its `KlasaMessage`.